devel/Dockerfile

```diff
@@ -57,6 +58,23 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
-ENV PATH="/opt/phoenix/utils:${PATH}"
+# install Jlink tools for 32-bit armv7 architecture
+RUN if [ "$TARGETPLATFORM" = "linux/arm/v7" ]; then \
+        mkdir -p /opt/SEGGER && \
+        (cd /opt/SEGGER && \
+        wget --post-data "accept_license_agreement=accepted" https://www.segger.com/downloads/jlink/JLink_Linux_arm.tgz && \
+        tar xf JLink_Linux_arm.tgz && \
+        chmod a-w JLink_Linux_V* && \
+        echo "$(ls -d JLink_Linux_V*)" > version && \
+        mv JLink_Linux_V* JLink_Linux_arm && \
+        mkdir -p /etc/udev/rules.d && \
+        cp JLink_Linux_arm/99-jlink.rules /etc/udev/rules.d/99-jlink.rules); \
+    fi
+
+ENV PATH="/opt/phoenix/utils:/opt/SEGGER/JLink_Linux_arm:${PATH}"
+
+#Provide script for loading files using gdb
+RUN mkdir -p /opt/phoenix/scripts
```

Comment:
this will create 2 separate docker overlays, if needed please merge with `install hostutils` step (IMHO, should not be needed gdb script should reside in project - not in docker)

Reply:
Yeah that's right, it won't be needed.